service/plugin/src/main/java/com/solace/maas/ep/event/management/agent/plugin/command/model/SempDeleteCommandConstants.java

```diff
@@ -0,0 +1,7 @@
+package com.solace.maas.ep.event.management.agent.plugin.command.model;
+
+public class SempDeleteCommandConstants {
+    public static final String SEMP_DELETE_ENTITY_TYPE = "entityType";
+    public static final String SEMP_DELETE_DATA = "data";
+    public static final String SEMP_DELETE_OPERATION = "delete";
```

Comment:
This is an utility/ constant class, please do the following
- make this class `final`
- add a private constructor which throws an `UnsupportedOperationException`